app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -364,5 +364,12 @@ data class Patient(
         AND syncStatus == 'DONE'
     """)
     abstract fun purgePatientAfterRetentionTime(now: Instant)
+
+    @Transaction
+    @Query("""
+      SELECT * FROM Patient
+      WHERE uuid = :patientUuid
+    """)
+    abstract fun contactPatientProfileImmediate(patientUuid: UUID): ContactPatientProfile
```

Comment:
Can we make this nullable similar to `patientProfileImmediate` function?

Reply:
So with contactPatientProfile, I don't see a scenario where this can be null?

Reply:
Hmm, good point. We expect patient to be present when calling this method. So I guess it should be fine.